Microsoft.HealthVault.UnitTest/ItemTypes/ApproximateDateTimeTests.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.HealthVault.ItemTypes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NSubstitute;
```

Comment:
Missing copyright header: 

// Copyright(c) Microsoft Corporation.
// This content is subject to the Microsoft Reference Source License,
// see [http://www.microsoft.com/resources/sharedsource/licensingbasics/sharedsourcelicenses.mspx](http://www.microsoft.com/resources/sharedsource/licensingbasics/sharedsourcelicenses.mspx).
// All other rights reserved.